packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -142,6 +177,7 @@ export interface _ContainerSummaryViewProps extends ContainerStateMetadata, ICon
 export function _ContainerSummaryView(props: _ContainerSummaryViewProps): React.ReactElement {
```

Comment:
We should be able to collapse this logic back into `ContainerSummaryView` now, right?

Reply:
Yes! Hadn't gotten around to that but the underscore views will all go away, merged into their non-underscore versions.